internal/runner/rspec_test.go

```diff
@@ -1,12 +1,71 @@
 package runner
 
 import (
+	"errors"
 	"os"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/kballard/go-shellquote"
 )
 
+func TestCommandNameAndArgs_WithInterpolationPlaceholder(t *testing.T) {
+	rspec := Rspec{}
+	testCases := []string{"spec/models/user_spec.rb", "spec/models/billing_spec.rb"}
+	testCommand := "bin/rspec --options {{testExamples}} --format"
+
+	gotName, gotArgs, _ := rspec.commandNameAndArgs(testCases, testCommand)
```

Comment:
Check the error return (mandatory), and ideally log the args to the function (highly recommended). This will help cut down investigation time when diagnosing possible test failures.

```suggestion
	gotName, gotArgs, err := rspec.commandNameAndArgs(testCases, testCommand)
	if err != nil {
		t.Errorf("Rspec.commandNameAndArgs(%q, %q) error = %v", testCases, testCommand, err)
	}
```

Reply:
ohh wow I totally forgot to check error return for those tests 😅 Thanks for calling it out! I'll apply the changes!